From e372c8400c2a7c59d9f3346a0e17a3dd46fc6c79 Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Mon, 21 Feb 2022 09:30:13 +0200
Subject: [PATCH] [SW-75471] gaudi2: security: renaming the hbw rr enum to be meaningful

Using HL_SEC_HBW_ICCM0_TO_RAZWI which has same value as
HL_SEC_HBW_ICCM0_TO_SP_SRAM to be meaningful in code walk through.

Change-Id: I0e11bbe16add8e8e2fb64f907578e45f0abac902
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 5a4f184..5d3835d 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -988,11 +988,13 @@
 	 * For noc and psoc rrs, need to protect from SPI_FLASH_END to DBG_SPACE_BASE
 	 * SPI FLASH END to DBG_SPACE_BASE
 	 * Reusing the HL_SEC_HBW_ICCM0_TO_SP_SRAM rr idx to increase the protection range.
+	 * Using HL_SEC_HBW_ICCM0_TO_RAZWI which has same value as HL_SEC_HBW_ICCM0_TO_SP_SRAM
+	 * to be meaningful in code walk through.
 	 * 0x1000007FF7000000 - 0x1000007FF8000000ull
 	 */
 	min = SPI_FLASH_END;
 	max = DBG_SPACE_BASE;
-	hl_secure_hbw_rr(HL_SEC_HBW_ICCM0_TO_SP_SRAM, min, max, rr_sel);
+	hl_secure_hbw_rr(HL_SEC_HBW_ICCM0_TO_RAZWI, min, max, rr_sel);
 
 	hl_secure_hbw_common(rr_sel);
 }
